hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/rbfbalance/RouterDistCpProcedure.java

```diff
@@ -44,6 +44,7 @@ protected void disableWrite(FedBalanceContext context) throws IOException {
     Configuration conf = context.getConf();
     String mount = context.getMount();
     MountTableProcedure.disableWrite(mount, conf);
+    updateStage(Stage.FINAL_DISTCP);
```

Comment:
Is there a test we can have for this?

Reply:
Thanks for your review. For now, there is no test class for this class. I'll consider adding a test class for this.